changelog/7772.feature.rst

```diff
@@ -0,0 +1,2 @@
+Added a new option ``autoalign='reproject'`` to :meth:`~sunpy/map/GenericMap/plot`.
+This option allows for automatic reprojection of a map's data to match the WCS of the target axes, using the :meth:`~sunpy/map/GenericMap/reproject_to` method.
```

Comment:
This is not how the interlinks work with sphinx unfortunately.

Reply:
Fixed them